pkg/integrations/discord/send-chat.go

```diff
@@ -54,11 +61,24 @@ func SendChat(ctx context.Context, w *discordtypes.Webhook, did string, scm *str
 
 	log.Warn(ctx, "sending chat to discord", "payload", string(jsonPayload))
 
-	req, err := http.NewRequestWithContext(ctx, "POST", w.URL, bytes.NewReader(jsonPayload))
+	port := parsedURL.Port()
+	if port == "" {
+		// Use default port based on scheme
+		if parsedURL.Scheme == "https" {
+			port = "443"
+		} else {
+			port = "80"
+		}
+	}
+	requestURL := fmt.Sprintf("%s://%s%s", parsedURL.Scheme, net.JoinHostPort(targetIP, port), parsedURL.RequestURI())
+
+	req, err := http.NewRequestWithContext(ctx, "POST", requestURL, bytes.NewReader(jsonPayload))
```

Comment:
Concerned about HTTPS here - a request to https://resolved_ip_address will fail on the TLS layer by default I think because the IP isn't in a SAN of the cert? Might need to go deeper on the TLS connection instantiation.

Or it just works but I'd be surprised, try it with a https://webhook.site URL